lstchain/tools/lstchain_create_sampling_interval_coefficient_file.py

```diff
@@ -0,0 +1,262 @@
+import numpy as np
+from astropy.io import fits
+import h5py
+from tqdm import tqdm
+
+from ctapipe.core import Provenance, traits
+from ctapipe.core import Tool
+from ctapipe_io_lst import LSTEventSource
+from ctapipe_io_lst.calibration import LSTR0Corrections
+from lstchain.calib.camera.sampling_interval_coefficient_calculate import SamplingIntervalCalculate
+from lstchain.paths import run_info_from_filename
+
+class SamplingIntervalCoefficientHDFWriter(Tool):
```

Comment:
This seems just to be three tools in one that don't really share code and even use different traitlets. I think really doing three smaller tools would be better and less confusing (since the help will only show the options for each step, not for all steps).

Reply:
I see. Actually, I just wanted to gather all of the procedures in a single file. Is it possible to put three smaller tools in a single file and call one of the tools when executing the command...? or one file per one tool?

Reply:
You can do multiple Tools per file and have different entrypoints, but I would just make three files